tests/e2e/common.go

```diff
@@ -67,14 +67,22 @@ var (
 	defaultDeployFluentbit = false
 )
 
-func DescribeTierN(tier uint) func(string, func()) bool {
-	return func(name string, body func()) bool {
-		return Describe(fmt.Sprintf("[Tier %d] %s", tier, name), body)
-	}
+type OSMDescribeInfo struct {
+	// tier represents the priority of the test. Lower value indicates higher priority.
+	tier int
+
+	// bucket indicates in which test bucket the test will run in for CI. Each
+	// bucket is run in parallel while tests in the same bucket run sequentially.
+	bucket int
```

Comment:
I wonder if it would be easier to understand what tests are grouped in what bucket if the buckets had a human-readable name (string).

Reply:
I used an `int` here just because I didn't want to come up with human-readable names and thought it would be a bit more resilient to typos (since we have to enumerate all the buckets in the CI config). I didn't come up with any sort of system to group the tests, so I'm not sure what a meaningful name would look like.